src/components/Buttons/docs/RadioCheckbox.md

```diff
@@ -0,0 +1,53 @@
+This component outputs a radio button or a checkbox.
+
+The programming logic behind producing these buttons is the same. The difference is in the props `btnType`. It defaults to radio if undefined.
+
+You can define whether the button starts in the pressed on unpressed state through the `isChecked` props.
+
+### Label
+Associating a label for every radio or checkbox button is very important for accessibility purposes. Use the `id` and `htmlFor` attributes to link them.
+
+There are times when it makes sense to have screenreaders say something different to what the label displays. For those cases, pass an aria label using the aria shape defined in the props.
+
+### Radio
+The component defaults to a radio button is a `btnType` is undefined. You can also pass the value `radio`. Also, you can pass a callback through the props `onClick` that returns both the click event and the checked/unchecked state.
+
+```js
+function onClick(event, isChecked) {
+  alert(`This button is ${isChecked ? '' : 'not '}checked`)
+}
+
+<div style={{display: 'flex', flexDirection: 'column'}}>
+  <div style={{display: 'flex', marginBottom: 20}}>
+    <RadioCheckbox id="radio1" onClick={onClick} />
+    <label htmlFor="radio1">Click me with callback</label>
```

Comment:
Would it make sense to allow users to pass a label directly in to the component? It seems like a common use case. Could be either a string or a rendered element, e.g. `<Checkbox label={<label>Make my profile visible</label>} />` (https://react.semantic-ui.com/modules/checkbox#checkbox-example-shorthand-element)

Reply:
The markup for the component would have to change slightly to something like this (pardon awkward spacing)

```
<div
        onChange={this.handleClick}
        onClick={this.handleClick}
      >
        <input
          checked={checked}
          name={name}
          type={type}
          value={value}
        />
        {*/ some kind of label (optional) */}
      </div>
```

Reply:
👍  taking a look now. this was mostly motivated by my assumption that most users of `RadioCheckbox` would have to create their own `label` element, which if true, means it makes sense for us to do that piece for them in snacks. I think it also lets us ensure labels always get applied because we're doing it at the snacks level. For the most part ripping directly from https://github.com/Semantic-Org/Semantic-UI-React/blob/master/src/modules/Checkbox/Checkbox.js#L222